lib/use-cases/create-conversation.js

```diff
@@ -1,23 +1,41 @@
 import { convertObjectToIsoDate } from 'lib/utils/date';
 
 export default class CreateConversation {
-  constructor({ conversationsGateway, sendSignpostEmail }) {
+  constructor({ conversationsGateway, sendSignpostEmail, sendSignpostLetter }) {
     this.conversationsGateway = conversationsGateway;
     this.sendSignpostEmail = sendSignpostEmail;
+    this.sendSignpostLetter = sendSignpostLetter;
   }
 
-  async execute({ resident, user, discussedServices, signPostingMessage, systemIds = [] }) {
+  async execute(
+    { resident, user, discussedServices, signPostingMessage, systemIds = [] },
+    sharingMethod
+  ) {
     resident.dateOfBirth = convertObjectToIsoDate(resident.dateOfBirth);
 
     const errors = [];
 
-    const signPostingEmail = await this.sendSignpostEmail.execute({
-      emailAddress: resident.email,
-      signPostingMessage
-    });
-
-    if (signPostingEmail.response.status > 201) {
-      errors.push('Failed to send sign posting email to resident');
+    let signPostingEmail;
+    if (sharingMethod == 'email') {
```

Comment:
I'd suggest enumerating this and other instances of letter/email, similar to the constants for Analytics events. Although less beneficial for Javascript over a typed language.